src/Application/Models/Bank/Asset.js

```diff
@@ -39,6 +40,14 @@ class Asset {
     this._multiplier += additionalPercentage;
   }
 
+  unlock() {
+    this._isUnlocked = true;
+  }
```

Comment:
I will make a ticket and think about an alternative. There has to be a way to signify that it should be rendered on the page...